app/models/host_status/configuration_status.rb

```diff
@@ -78,7 +78,7 @@ def to_status(options = {})
     end
 
     def relevant?
-      host.configuration?
+      host.configuration? || last_report.present?
```

Comment:
so this means new host without a last_report will never show up its configuration status without a proxy?